tests/testthat/test-parsers.R

```diff
@@ -0,0 +1,25 @@
+test_that("parse_errors_to_df parses correct assertion type and correct error_df", {
+  data <- data.frame(
+    V1 = c(1, 0),
+    V2 = c(-1, -2)
+  )
+  validation <- validate_if(data, V1 < 0)
+  parsed <- parse_errors_to_df(validation)
+
+  expect_equal(parsed$type, error_id)
+  expect_equal(nrow(parsed$error_df[[1]]), sum(!data$V1 < 0))
+})
+
+test_that("parse_successes_to_df parses success correctly", {
```

Comment:
Please provide a more specific test description.

Reply:
Test case & description modified in [33b781a](https://github.com/Appsilon/data.validator/pull/59/commits/33b781a8bec60c537c97cc06b36b88657146e67e).